### DISCUSSION TITLE
domain is in use by another Vercel account

### DISCUSSION THREAD

**Discussion by shirishpokhrelappsinvoofficial** [vercel/community#3470] [Help] [answered]
### Summary

I need to clear the domain from other account so that it can be used here. Also if I wish to use this domain in my future projects, how could this be done?

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/107414943/7b32e88e-eae4-4136-a498-4659295e5114)

**Comment by amyegan** [comment, marked as answer]
Hey @shirishpokhrelappsinvoofficial. If the other account belongs to you, then you can [move the domain](https://vercel.com/docs/concepts/projects/domains/transfer-your-domain#transfer-a-domain-to-another-vercel-user-or-team). Otherwise, simply follow the instructions to add the TXT record with your domain registrar and the domain will be connected to your account. 🙂
